### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: build/build-all.sh [hunk 1]
```diff
@@ -18,9 +18,6 @@ $SCRIPT_DIR/build-ios.sh -c Release -b $WORKSPACE/derivedData -f SalesforceSDKCo
 $SCRIPT_DIR/build-ios.sh -c Debug -b $WORKSPACE/derivedData -f SmartStore
 $SCRIPT_DIR/build-ios.sh -c Release -b $WORKSPACE/derivedData -f SmartStore
 
-$SCRIPT_DIR/build-ios.sh -c Debug -b $WORKSPACE/derivedData -f SmartStore
```
Comment (bhariharan) [inline build/build-all.sh]: Removed duplicate entry.

File: external/cordova-ios/sfdc_build/build.xml [hunk 2]
```diff
@@ -1,98 +0,0 @@
-<!--
-    Build file for CordovaLib
```
Comment (bhariharan) [inline external/cordova-ios/sfdc_build/build.xml]: This is `circa-2012` :-)

**Comment by ivanbogdanov** [comment]
Looks good to me

**Comment by huminzhi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Code coverage under minimum of 80%</td>
    </tr>
  </tbody>
</table>

### Tests: 

Classname | Name|
--- | ---|
MetadataManagerTests | testCommonMRUObjectsFromServer|
MetadataManagerTests | testCommonMRUObjectsFromServer|
SyncManagerTests | testCleanResyncGhostsForMRUTarget|


## Current coverage for SalesforceAnalytics.framework is `72%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceSDKCore.framework is `51%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartStore.framework is `63%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSync.framework is `76%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceHybridSDK.framework is `43%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceReact.framework is `0%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_CI-result">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Pull request merged**